clyent/__init__.py

```diff
@@ -96,7 +96,9 @@ def add_default_arguments(parser, version=None):
     ogroup.add_argument('--no-color', action='store_const', dest='color',
                         const='never',
                         help='never display with colors')
-
+    ogroup.add_argument('-j', '--json',
```

Comment:
I don't think we need a short option here. This should only be used as a programmatic interface.